### PR TITLE
[PREVIEW] Sidam

### PR DESCRIPTION
**Pull request by linusnorton**
SIDAM it all to hell.

### CODE REVIEW

File: docker-compose-dependencies.yml [hunk 1]
```diff
@@ -2,8 +2,18 @@
 version: '2.1'
 
 services:
+
+  rpa-native-pdf-annotator-db:
```
Comment (JeroenRijks) [inline docker-compose-dependencies.yml]: I agree with moving the app's db in here. it's a dependency, doesn't belong in docker-compose.yml

File: docker-compose-dependencies.yml [hunk 2]
```diff
@@ -86,48 +97,57 @@ services:
       - dm-store-db
       - service-auth-provider-app
     ports:
-     - 4603:8080
+      - 4603:8080
 
   dm-store-db:
-    image: hmcts/dm-store-db:latest
+    image: postgres
```
Comment (JeroenRijks) [inline docker-compose-dependencies.yml]: aren't they using Blob store?

Comment (linusnorton) [reply docker-compose-dependencies.yml]: unsure

File: docker-compose-dependencies.yml (hunk 1 above)
Comment (linusnorton) [reply docker-compose-dependencies.yml]: it's required when you run the dependencies, otherwise you'd have to spin up docker twice.

**Pull request merged**